### ISSUE TITLE
The test section is not found in the supplied caption data

### ISSUE THREAD

**Issue by yjmlaile** [salanueva/CBM#1]
![1704198542371](https://github.com/salanueva/CBM/assets/31871582/bdb3cb9c-8163-4863-b035-451d92abc084)
The caption data is provided in the code, but there are only training sets and verification sets, and the model needs to load test data during training, which leads to the inability to train. The caption of the test set can not be found online, hope to provide it.
